packages/nimble/R/genCpp_sizeProcessing.R

```diff
@@ -252,6 +252,8 @@ sizeDim <- function(code, symTab, typeEnv) {
 }
 
 sizeDiagonal <- function(code, symTab, typeEnv) {
+    ## refactor: code$name change step stays here
```

Comment:
nit: If you label your comments `useRefactoredSizeProcessing`, you will have an easier time systematically cleaning up those comments when you're done.